misaligned.js

```diff
@@ -2,13 +2,28 @@ const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
 const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
 
 function print_color_map() {
-    let color_map = "";
-    for (let i = 0; i < majorColors.length; i++) {
-        for (let j = 0; j < minorColors.length; j++) {
-            color_map += `${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}\n`;
-        }
+  let color_map = "";
+  //max length of index is calculated
+  const maxIndexWidth = String(majorColors.length * minorColors.length).length;
+  //max length of majorcolor is calculated

```

Comment:
Nice approach to adapt based on the max length :+1: 
Same code will work, even when the colors are translated to different languages, where the words may be longer